processors/ratelimiter/ratelimiterprocessor.go

```diff
@@ -48,8 +48,8 @@ type processor struct {
 }
 
 const (
-	TenantID       = "tenant_id"
-	GlobalTenantID = "global_tenant_id"
+	TenantSpans  = "tenant_spans"
```

Comment:
we dont want tenant id?

Reply:
Tenant-id is needed, updated name which is aligned with spans